src/nnet3bin/nnet3-copy.cc

```diff
@@ -91,8 +94,11 @@ int main(int argc, char *argv[]) {
     if (scale != 1.0)
       ScaleNnet(scale, &nnet);
     
-    if (dropout > 0)
-      SetDropoutProportion(dropout, &nnet);
+    if (dropout >= 0)
+      KALDI_ERR << "--dropout option is deprecated. "
```

Comment:
You can just remove the dropout option from the code; it's not used in any example scripts.  Only people here at JHU were probably using it and they can easily ask around or check the git history.